assets/src/web-stories-block/css/variables.css

```diff
@@ -0,0 +1,17 @@
+:root {
+  --ws-font-size-title: 18px;
+  --ws-font-size-circle-title: 14px;
+  --ws-font-size-excerpt: 16px;
+  --ws-font-size-author-date: 14px;
+  --ws-aspect-ratio: 176.2%;
+  --ws-list-aspect-ratio: 75%;
+  --ws-overlay-text-color: #fff;
+  --ws-overlay-text-lh: 1.3;
+  --ws-story-min-width: 185px;
+  --ws-circle-size: 96px;
+  --ws-circle-border-color: #ccc;
+}
+
+/* Media Query */
+$ws-mq-large: 676px; /* Large as above this we don't need any other breakpoint. */
+$ws-mq-medium: 415px; /* For styles not meant for mobile/smaller screens, like columns view. */
```

Comment:
Why not use css variables as above?

Reply:
media queries doesn't support `var()` yet.